tests/unit/test_graph.py

```diff
@@ -0,0 +1,71 @@
+import unittest
```

Comment:
Nice, al hoeft wat mij betreft een 'added' graph niet een nieuwe naam te krijgen.

Reply:
Ik dacht: makkelijk om het te onderscheiden en bovendien heeft de naam nu een setter, dus kun je 'm zo wijzigen. En wat is het alternatief? `g + h` levert een graaf op, maar welke naam geef je 'm dan? Default? Leeg? Of wat ik nu heb gedaan. Het is denk ik om het even.

Reply:
Oh ik kijk meestal meer naar de vertices ipv de graaf zelf.